front/src/applications/operationalStudies/views/v2/ScenarioV2.tsx

```diff
@@ -113,7 +114,11 @@ const ScenarioV2 = () => {
   );
 
   const { data: conflicts } = osrdEditoastApi.endpoints.getV2TimetableByIdConflicts.useQuery(
-    { id: timetable?.id as number, infraId: infraId! },
+    {
+      id: timetable?.timetable_id as number,
+      infraId: infraId!,
+      electricalProfileSetId: scenario?.electrical_profile_set_id,
+    },
     { skip: !timetable || !infraId }
```

Comment:
To harmonize with other params, we should either : 
- get electricalProfilSetId with a useSelector and use it in the endpoint and add `!electricalProfilSetId` in `skip`

```suggestion
      const { data: conflicts } = osrdEditoastApi.endpoints.getV2TimetableByIdConflicts.useQuery(
    {
      id: timetable?.timetable_id as number,
      infraId: infraId!,
      electricalProfileSetId: electricalProfileSetId,
    },
    { skip: !timetable || !infraId || !electricalProfileSetId }
  );
```

- chose to get all these infos from the scenario as it carries all we need

```suggestion
    const { data: conflicts } = osrdEditoastApi.endpoints.getV2TimetableByIdConflicts.useQuery(
    {
      id: scenario?.timetable_id as number,
      infraId: scenario?.infra_id as number,
      electricalProfileSetId: scenario?.electrical_profile_set_id,
    },
    { skip: !scenario }
  );
```

Reply:
I picked your second option, but used `!` instead of `?`, what do you think ,